tests/functional/splinter_test.c

```diff
@@ -2572,10 +2579,16 @@ splinter_test(int argc, char *argv[])
    /*
     * 2. Parse test_config options, see test_config_usage()
     */
+
    test_config *test_cfg = TYPED_ARRAY_MALLOC(hid, test_cfg, num_tables);
    for (uint8 i = 0; i < num_tables; i++) {
       test_config_set_defaults(test, &test_cfg[i]);
+
+      // All tables share the same message-generator.
       test_cfg[i].gen = &gen;
+
+      // And, all tests & tables share the same test-execution parameters.
+      test_cfg[i].test_exec_cfg = &test_exec_cfg;
```

Comment:
Minor rearrangement of data structures.

Poke ptr to test execution config struct in `test_cfg`, ... so that we can get at the new `verbose_progress` boolean parameter.